onadata/apps/logger/models/attachment.py

```diff
@@ -74,7 +74,7 @@ def secure_url(self, suffix="original"):
         if suffix != "original" and suffix not in settings.THUMB_CONF.keys():
             raise Exception("Invalid image thumbnail")
 
-        return u"{kobocat_url}{media_url}{suffix}?media_file={filename}".format(
```

Comment:
This code also exists in a management command.
https://github.com/kobotoolbox/kobocat/blob/c17e41d0314c354882164993715ac5272f46ca1c/onadata/apps/viewer/management/commands/rewrite_attachments_urls_in_mongo.py#L106-L110.
Would you mind keep the same variable name in both places?
Either you change here `filename` to `media_file`, either you change `media_file` to `filename` in the management command.